DataFormats/HGCalReco/src/classes_def.xml

```diff
@@ -55,4 +55,58 @@
   <class name="std::vector<TICLCandidate>" />
   <class name="edm::Wrapper<TICLCandidate>" />
   <class name="edm::Wrapper<std::vector<TICLCandidate> >" />
+  <class  name="HGCalSoACells"/>
+
+  <class name="HGCalSoACellsHostCollection"/>
+  <!-- Read streamer declaration for dictionary. It ensures allocation of buffer and data copying to it -->
+  <read
+    sourceClass="HGCalSoACellsHostCollection"
+    targetClass="HGCalSoACellsHostCollection"
+    version="[1-]"
+    source="HGCalSoACells layout_;"
+    target="buffer_,layout_,view_"
+    embed="false">
+  <![CDATA[
+    HGCalSoACellsHostCollection::ROOTReadStreamer(newObj, onfile.layout_);
+  ]]>
+  </read>
+  <class name="edm::Wrapper<HGCalSoACellsHostCollection>" splitLevel="0"/>
```

Comment:
Please add a class version, and move the read rules to `DataFormats/HGCalReco/src/classes.cc`.

Here you should keep just:
```suggestion

  <class name="HGCalSoACells"/>
  <class name="HGCalSoACellsHostCollection" ClassVersion="3">
    <version ClassVersion="3" checksum="........"/>
  </class>
  <class name="edm::Wrapper<HGCalSoACellsHostCollection>" splitLevel="0"/>
```

In `DataFormats/HGCalReco/src/classes.cc` you should use:
```c++
#include "DataFormats/EcalRecHit/interface/EcalUncalibratedRecHitHostCollection.h"
#include "DataFormats/Portable/interface/PortableHostCollectionReadRules.h"
   
SET_PORTABLEHOSTCOLLECTION_READ_RULES(HGCalSoACellsHostCollection);
```

Reply:
Same for `HGCalSoACellsOutHostCollection` and `HGCalSoAClustersHostCollection`.